### ISSUE TITLE
How to address a vary basic question: which tools at hand to check if a series candidate for SA

### ISSUE THREAD

**Issue by andreranza** [christophsax/seasonalbook.content#21]
Maybe an idea for an exercise to consolidate a fundamental question that might still be forgotten. 
I realized it on myself when I started to call `seas()` like a monkey without even asking myself whether it made sense what I was trying to do.
I don't know where this would fit best. This, for sure, is part of the "getting started" section.
Before starting with `seas()`, which tools can I use to explore a series, especially in case where the seasonal pattern is not so evident with eye-balling?
It would be nice to have a dedicated small section listing all the tools needed to squeeze a series before starting modeling:

- `plot(ts)`
- `plot(decompose(ts))`
- `plot(stl(ts, s.window = "periodic"))`
- `monthplot(ts)`
- `acf(ts)`
- `qs(seas(ts))`
